scripts/library.sh

```diff
@@ -437,3 +428,59 @@ function lint_markdown() {
   # https://github.com/markdownlint/markdownlint
   run_lint_tool mdl "linting markdown files" "-r ~MD013" $@
 }
+
+# Call a function and verify its return value and output.
+# Parameters: $1 - expected return code.
+#             $2 - expected output ("" if no output is expected)
+#             $3 ..$n - function to call and its parameters.
+function test_function() {
```

Comment:
Let's not add it here, since this is test-infra unit-test specific. Please create a separate helper script inside `test/unit`.

Reply:
done, please review